test/common/tracing/http_tracer_impl.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package test.common.tracing;
+
+import "envoy/type/tracing/v2/custom_tag.proto";
+
+message CustomTagCases {
```

Comment:
In general we only use a proto for a test if we are going to throw it into a fuzzer. I don't see the need for it here. Can you just define a standard struct, etc. in the test you need it for?